v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/options/ElasticsearchWriteOptions.java

```diff
@@ -73,4 +73,49 @@ public interface ElasticsearchWriteOptions extends PipelineOptions {
   Long getMaxRetryDuration();
 
   void setMaxRetryDuration(Long maxRetryDuration);
+
+  @Description("Gcs path to javascript udf source for index extraction function")
```

Comment:
Please make the changes requested in PR #399 here too (e.g. change parameter descriptions) and let me know when done, I'll do another pass.

Reply:
@an2x ok, I've rebased this PR against the lastest main (that has the merged changes from the PR #399